nemo/core/classes/modelPT.py

```diff
@@ -77,6 +85,39 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
             if 'test_ds' in self._cfg and self._cfg.test_ds is not None:
                 self.setup_test_data(self._cfg.test_ds)
 
+    def register_artifact(self, conf_path: str, src: str):
+        """
+        Register model artifacts with this function. These artifacts (files) will be included inside .nemo file
+        when model.save_to("mymodel.nemo") is called.
+
+        WARNING: If you specified /example_folder/example.txt but ./example.txt exists, then ./example.txt will be used.
+
+        Args:
+            conf_path: config path where artifact is used
+            src: path to the artifact
```

Comment:
Lets use `config_path` instead of `conf_path`, as mentioned above.

Reply:
changed